starlite/handlers/http.py

```diff
@@ -268,6 +270,7 @@ def __init__(
         operation_id: Optional[str] = None,
         raises: Optional[List[Type[HTTPException]]] = None,
         response_description: Optional[str] = None,
+        responses: Optional[Dict[int, AdditionalResponse]] = None,
```

Comment:
could this be called `additional_responses`?

Reply:
in the OA 3.1 specs this is called responses afaik. please verify this @seladb by looking at the `pydantic-openapi-schema` lib.

Reply:
And if so, very happy to leave it as it is.

Reply:
it is: https://spec.openapis.org/oas/latest.html#responses-object

And the key in `operation` is called `responses`: https://spec.openapis.org/oas/latest.html#operation-object

Reply:
So we should leave it as it, right?
BTW, this is also how it's called in FastAPI...

Reply:
Yep, leave it.